compiler/rustc_lint/src/builtin.rs

```diff
@@ -3147,6 +3147,7 @@ declare_lint! {
     /// ### Example
     ///
     /// ```rust,compile_fail
+    /// # #![feature(asm)]
```

Comment:
This shouldn't be necessary.  The asm feature was stabilized in the 1.59 release.

Reply:
Got it!